engine/crates/federated-graph/src/render_sdl.rs

```diff
@@ -585,36 +583,103 @@ impl Display for Description<'_> {
     }
 }
 
+// Copy-pasted from async-graphql-value
+fn write_quoted(sdl: &mut impl Write, s: &str) -> fmt::Result {
+    sdl.write_char('"')?;
+    for c in s.chars() {
+        match c {
+            '\r' => sdl.write_str("\\r"),
```

Comment:
```suggestion
            c @ '\r' | '\n' | '\t' | '"' | '\\' => {
                sdl.write_char('\\');
                sdl.write_char(c);
            }
```

Reply:
And without the few further match arms. 🙂

Reply:
indeed good catch